src/utils.ts

```diff
@@ -50,6 +50,7 @@ export function parseDataNft(value: NftType): DataNft {
     royalties: value.royalties / 100,
     nonce: value.nonce,
     collection: value.collection,
+    balance: value.balance ? Number(value.balance) : 0,
```

Comment:
It shouldn't be supply instead of balance ? @icegriffinguru

Reply:
@icegriffinguru @bucurdavid yes, we should use terminology that's the same everywhere. In the Data DEX, where / what is this attribute called in the back end/ api?

<img width="278" alt="image" src="https://github.com/Itheum/sdk-mx-data-nft/assets/743169/35651e57-f8f3-43a2-97a4-e8c31a8a9d21">

Reply:
`balance` and `supply` are different terms.
`balance` is what you are owning and `supply` is total number of issued amount of that nonce. @bucurdavid